docs/guides.rst

```diff
@@ -67,8 +67,9 @@ You can also find out which input files are required for a given preparation sta
      - free_preequil.prm7, free_preequil.rst7
      - The solvated ligand after heating and short initial equilibration steps
 
-In addition, for every preparation stage, **template_config.cfg must be present in the input
-directory.**
+The default simulation engine is SOMD ``engine_config = a3.SomdConfig()``,the details of its configuration could be customised freely.
+For example, ``engine_config = a3.SomdConfig(constants="hbonds", cutoff_type="PME", integrator="langevinmiddle", thermostat=False...)``
+But you can also use Gromacs ``engine_config = a3.GromacsConfig()``.
```

Comment:
Please remove this line - we'll add in when we add the gromacs functionality. Thanks